addons/web/static/tests/views/fields/numeric_fields_tests.js

```diff
@@ -306,4 +310,64 @@ QUnit.module("Fields", (hooks) => {
             });
         }
     );
+
+    QUnit.test(
+        "useNumpadDecimal should synchronize handlers on input elements",
+        async function (assert) {
+            target = getFixture();
+            patchWithCleanup(localization, { decimalPoint: ",", thousandsSep: "." });
```

Comment:
these two lines are not necessary:
- target is already set in the beforeEach
- the localization service is set in the beforeEach and will already patch the localization data with the same values